geograypher/meshes/meshes.py

```diff
@@ -851,6 +861,17 @@ def get_values_for_verts_from_vector(
             return labeled_verts, all_values
         # Else return a dict of all requested values
         return labeled_verts_dict, all_values_dict
+    
+    def save_IDs_to_labels(self, savepath: PATH_TYPE):
+        savepath = Path(savepath)
```

Comment:
I'd recommend ensuring the containing directory exists before saving the file.

Reply:
Noted, we discussed how mkdir doesn't influence already existing directories if exist_ok is True. So we can add mkdir to save_IDs_to_labels.